client/src/components/R&R/ReviewsBreakdown.jsx

```diff
@@ -35,19 +35,22 @@ const ReviewsBreakdown = () => {
 
   const starPercents = {};
   const calcRatingPercentages = () => {
-    const keys = [];
+    const keys = ['5', '4', '3', '2', '1'];
     let ratingTotal = 0;
-    Object.keys(reviewsMeta.ratings).forEach((key) => {
-      keys.push(key);
-    });
     keys.sort((a, b) => {
       return b - a;
     });
     keys.forEach((key) => {
-      ratingTotal += Number(reviewsMeta.ratings[key]);
+      if (Number(reviewsMeta.ratings[key]) > 0) {
+        ratingTotal += Number(reviewsMeta.ratings[key]);
+      }
     });
```

Comment:
This seems like a prime opportunity for reduce, if one were inclined to witchcraft